core/sqf/sql/scripts/get_libhdfs_files

```diff
@@ -37,10 +42,11 @@ fi
 LOGFILE=${LIBHDFS_TEMP_DIR}/build.log
 
 # Hadoop source tar file to build libhdfs from
-HADOOP_SRC_MIRROR_URL=https://archive.apache.org/dist/hadoop/common/hadoop-2.6.0
+HADOOP_MIRROR_URL=https://archive.apache.org/dist/hadoop/common/hadoop-2.6.0
```

Comment:
Would it make sense to replace the 2.6.0 with ${HADOOP_DEP_VER}? Just in case this script survives for a longer time.